src/Microsoft.Extensions.Configuration.Xml/EncryptedXmlDocumentDecryptor.cs

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-#if !DOTNET && !DNXCORE50
+#if !DOTNET5_4 && !DOTNET5_4
```

Comment:
Yes. I have file bug https://github.com/aspnet/Configuration/issues/317 to clean this up and make sure we only have the #ifs we need. 

Keep in mind that this was just a quick fix for the build. (Thanks @pranavkm!)